app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
         minSdkVersion 21
         targetSdkVersion 29
         versionCode 22
-        versionName '3.0.4'
+        versionName '3.0.5'
```

Comment:
revert, please. need to keep the version number provided whenever you rebase on sillsdev:develop